### PR TITLE
Subscribe satellite as the ddns package requires java package

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
There is an update in the ddns package url, which needs the java and causing failure of the job.  see the custom cert job 186 it is fixed with this line of code change. The remaining code needs to be updated in the gitlab repo.

### CODE REVIEW

File: automation_tools/__init__.py
Comment (devendra104) [inline automation_tools/__init__.py:175]: It will work, but hostname would be better instead of IP.

**Review by devendra104 — APPROVED** [review]
Ack.

File: automation_tools/__init__.py
Comment (omkarkhatavkar) [reply automation_tools/__init__.py:175]: I tried that earlier and failed with `AttributeError: 'NoneType' object has no attribute 'strip'`. please take a look 185 job output

**Review by jyejare — APPROVED** [review]
ACK pending required changes in CI.

**Comment by jyejare** [comment]
@omkarkhatavkar Until you add the update the code in CI dont merge this PR, else it will fail the CI runs. Mark your PR `draft`.

**Comment by omkarkhatavkar** [comment]
@jyejare no required changes in CI

**Comment by jyejare** [comment]
As per discussion with @omkarkhatavkar , CI MR is already open and after reviewing and merging that PR , merging this PR with no know side effects.

**Pull request merged**
